packages/venia-concept/src/components/Input/input.js

```diff
@@ -0,0 +1,139 @@
+import { Component, createElement } from 'react';
```

Comment:
This input has a lot of things that the `informed` input does not. I like it a lot; however, if you can refactor it so it is based on the `informed` input, I would feel better about merging it. Let me know how you want to proceed.

Reply:
So I implemented 'Form' from Informed like I mentioned above, and that was great, but the `Text` component of Informed (their version of `input`) doesn't seem to add anything here, and it actually makes some things more cumbersome and less readable. 

For example, I built the feature where the email/username you enter on signIn gets carried over to the createAccount input, but Informed's `Text` does not play well with `defaultValue`. Their version, `initialValue`, is loaded on component mount which makes the aforementioned feature much more difficult since the email/username is empty on mount. And I can't have a `defaultValue` with a `field`, which is frustrating because `field` is required.

I'm sure I could get this to work, but in the end it doesn't seem to add anything for the hassle.

If you would like, I can revisit this Monday. I'm happy to take another look.

Reply:
It looks like using Informed's <Text /> component actually makes some features more difficult (initialValue, for example, loads on mount so it doesn't allow us to easily copy the email a user enters from the `login` form to the `create account` form without doing some funky stuff).

I understand the appeal of using it, but @pcvonz and I are finding it to just be cumbersome at this point. Is it OK to omit using it for now?